math/slepc/Portfile

```
@@ -22,27 +22,36 @@ notes                       "Add the following lines to your .bash_profile if yo
                             \texport PETSC_DIR=${prefix}/lib/petsc\n\
                             \texport SLEPC_DIR=${prefix}/lib/slepc"
 
-platforms                   darwin
-
 homepage                    https://slepc.upv.es
 master_sites                https://slepc.upv.es/download/distrib/
 
-checksums                   rmd160  76c818bd5e4cd14b7c738d5f765592316a77d409 \
-                            sha256  e53783ae13acadce274ea65c67186b5ab12332cf17125a694e21d598aa6b5f00 \
-                            size    5191730
+checksums                   rmd160  032ee36fec86fc97b68b9de1e5f3ad6bff00d082 \
+                            sha256  beb33f0a15c3ce81744b15ad09ddf84dae70dbf3475c5ef032b8549ab87d6d8a \
+                            size    1827860
 
 mpi.setup                   default
 mpi.enforce_variant         petsc
 
 depends_lib-append          port:petsc
 use_parallel_build          no
 
-set python.version          39
-set python_ver_dot          [join [split ${python.version} {}] "."]
+set python_ver_dot          3.11
+set python.version          [string map {. ""} $python_ver_dot]
 
 depends_build-append        port:sowing \
                             port:python${python.version}
 
+configure.python            ${prefix}/bin/python${python_ver_dot}
+
+post-patch {
+    reinplace "s|/usr/bin/env python3|${configure.python}|" ${worksrcpath}/configure
+    fs-traverse f ${worksrcpath} {
```

The second argument to `fs-traverse` must be a list of paths:

```suggestion
    fs-traverse f [list ${worksrcpath}] {
```